templates/nginx/php-fpm/fossbilling.tpl

```diff
@@ -35,9 +35,11 @@ server {
         }
     }
     
-    # Disable PHP execution in bb-uploads and bb-data
-     location ^~ /uploads/ { }
-     location ^~ /data/ {
+    # Disable PHP execution in fb-uploads and fb-data
+     location ^~ /uploads/.*\.php$ { 
+       deny all;   
+     }
+     location ^~ /data/.*\.php$ {
```

Comment:
This makes *.log files also public 

Something like should prevent that
location ^~ /data/logs/ {
return 401; 
}

Not Tested TM

Reply:
good point, I added lines to block access for /data/log/ and /data/cache/
also added a missing line to block access to sensitive files